Sources/PortalSwift/PortalTypes.swift

```diff
@@ -0,0 +1,148 @@
+public struct AnyEncodable: Encodable {
```

Comment:
Types from the `Portal.swift` file have been moved here for organization purposes